### ISSUE TITLE
Test failure in TraversalInterruptionComputerTest

### ISSUE THREAD

**Issue by sjudeng** [JanusGraph/janusgraph#258]
Observing a test failure when isolating `TraversalInterruptionComputerTest#shouldRespectThreadInterruptionInVertexStep` through ThriftComputerTest.

```
shouldRespectThreadInterruptionInVertexStep[expectInterruption(g_V_out)](org.apache.tinkerpop.gremlin.process.traversal.TraversalInterruptionComputerTest)  Time elapsed: 4.511 sec  <<< FAILURE!
java.lang.AssertionError: 
Expected: is <true>
     but: was <false>
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.junit.Assert.assertThat(Assert.java:956)
	at org.junit.Assert.assertThat(Assert.java:923)
	at org.apache.tinkerpop.gremlin.process.traversal.TraversalInterruptionComputerTest.shouldRespectThreadInterruptionInVertexStep(TraversalInterruptionComputerTest.java:100)
```

Issue appears to be associated with the TinkerPop 3.2.4 update as it passes under 3.2.3. Test passed in full test runs as part of #208.

**Comment by amcp** [comment]
Also failing on hbase098 TP tests. Checking hbase10 now

**Comment by amcp** [comment]
HBase10 did not appear to fail but I am running them again to be sure.

**Comment by sjudeng** [comment]
This issue is resolved by https://github.com/apache/tinkerpop/pull/630 but this won't be merged in time for TinkerPop 3.2.5.
